cms/static/js/certificates/views/certificate_details.js

```diff
@@ -0,0 +1,71 @@
+// Backbone Application View: Certificate Details
+
+define([
+    'js/views/baseview', 'underscore', 'js/certificates/models/signatory', 'js/certificates/views/signatory_details', 'gettext', 'underscore.string'
+],
+function(BaseView, _, Signatory, SignatoryDetails, gettext, str) {
+    'use strict';
+    var CertificateDetailsView = BaseView.extend({
+        tagName: 'div',
+        events: {
+            'click .edit': 'editCertificate',
+            'click .show-details': 'showDetails',
+            'click .hide-details': 'hideDetails'
+        },
+
+        className: function () {
+            // Determine the CSS class names for this model instance
+            var index = this.model.collection.indexOf(this.model);
+
+            return [
+                'collection',
+                'certificates',
+                'certificate-details',
+                'certificate-details-' + index
+            ].join(' ');
+        },
+
+        initialize: function() {
+            // Set up the initial state of the attributes set for this model instance
+            this.template = this.loadTemplate('certificate-details');
+            this.listenTo(this.model, 'change', this.render);
+        },
+
+        editCertificate: function(event) {
+            // Flip the model into 'editing' mode
+            if (event && event.preventDefault) { event.preventDefault(); }
```

Comment:
If it's really a javascript `event` then `preventDefault` should be present - should do no harm, and allows one to call these functions with a "fake" event if desired (maybe for testing...)